planner/core/logical_plan_builder.go

```diff
@@ -3005,7 +3005,7 @@ func (b *PlanBuilder) buildUpdateLists(
 	error error,
 ) {
 	b.curClause = fieldList
-	modifyColumns := make(map[string]struct{}, p.Schema().Len()) // Which columns are in set list.
+	modifyColumns := make(map[string]bool, p.Schema().Len()) // Which columns are in set list.
```

Comment:
1:better move the comment to the top of the line.
2:here seems can still use `map[string]struct{}`, `true` write it into map, `false` do nothing.

Reply:
1. OK, good idea.
2. In [L3044](https://github.com/pingcap/tidb/pull/13211/files#diff-b9e75a99170ea35a67074d5d3d3305adR3044), We need both indicate if this column is a modified column and if the value assigned to this column is `DEFAULT`. So, keep we use `map[string]bool`

Reply:
Yeah, I mean, if you can find it in map, that means `DEFAULT`, if you can't find it / !ok, that means `NOT DEFAULT`.